packages/cbioportal-utils/src/oncokb/OncoKbUtils.ts

```diff
@@ -250,14 +250,18 @@ export enum StructuralVariantType {
 
 export function generateAnnotateStructuralVariantQueryFromGenes(
     site1EntrezGeneId: number,
-    site2EntrezGeneId: number,
+    site2EntrezGeneId: number | undefined,
```

Comment:
is it possible that site1 could be missing?  or by definition, no?  sorry if this is stupid question